<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPyCrypto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string>Choose a file to encrypt with menu file/encrypt, or enter text here. And click &quot;Encrypt&quot;.
Choisissez un fichier à chiffrer à l'aide du menu, Ou entrez votre texte ici puis cliquez sur chiffrer.</string>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="4">
        <widget class="QPushButton" name="pushDecrypt">
         <property name="text">
          <string>Decrypt (GnuPG)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushEncrypt">
         <property name="text">
          <string>Encrypt (GnuPG)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushSign">
         <property name="text">
          <string>Sign (GnuPG)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushVerify">
         <property name="text">
          <string>Verify (GnuPG)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitter"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionSauver"/>
   <addaction name="separator"/>
   <addaction name="actionVider_Tampon"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopier"/>
   <addaction name="actionColler"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfos">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfos_QPyCrypto"/>
    <addaction name="actionAbout_GNUPG"/>
    <addaction name="actionInfos_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>GnuPG</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChiffrerfichier"/>
    <addaction name="actionDechiffrerfichier"/>
    <addaction name="separator"/>
    <addaction name="actionSigner_un_fichier"/>
    <addaction name="actionVerifier_signature"/>
    <addaction name="separator"/>
    <addaction name="actionKeymanager"/>
   </widget>
   <widget class="QMenu" name="menuTampon">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauver"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
    <addaction name="separator"/>
    <addaction name="actionVider_Tampon"/>
    <addaction name="separator"/>
    <addaction name="actionChiffrer_OpenPGP"/>
    <addaction name="actionDechiffrer_OpenPGP"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuToolbar">
    <property name="title">
     <string>Toolbar</string>
    </property>
    <addaction name="actionCacher"/>
    <addaction name="actionShow_text_icons"/>
   </widget>
   <widget class="QMenu" name="menuEnigma">
    <property name="title">
     <string>Enigma</string>
    </property>
    <addaction name="actionChiffrer_un_fichier_Enigma"/>
    <addaction name="actionD_chiffrer_un_fichier_Enigma"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuHash">
    <property name="title">
     <string>Hash (not implemented)</string>
    </property>
    <addaction name="actionRIPEMD"/>
    <addaction name="actionSHA"/>
    <addaction name="actionSHA224"/>
    <addaction name="actionSHA256"/>
    <addaction name="actionSHA384"/>
    <addaction name="actionSHA512"/>
    <addaction name="actionMD2"/>
    <addaction name="actionMD4"/>
    <addaction name="actionMD5"/>
   </widget>
   <addaction name="menuEnigma"/>
   <addaction name="menuFichier"/>
   <addaction name="menuHash"/>
   <addaction name="menuTampon"/>
   <addaction name="menuToolbar"/>
   <addaction name="menuInfos"/>
  </widget>
  <action name="actionInfos_QPyCrypto">
   <property name="text">
    <string>About QPyCrypto</string>
   </property>
  </action>
  <action name="actionInfos_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionChiffrerfichier">
   <property name="text">
    <string>Encrypt file (GnuPG)</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icones/quit.png</normaloff>icones/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDechiffrerfichier">
   <property name="text">
    <string>Decrypt file (GnuPG)</string>
   </property>
  </action>
  <action name="actionRIPEMD">
   <property name="text">
    <string>RIPEMD</string>
   </property>
  </action>
  <action name="actionMD2">
   <property name="text">
    <string>MD2</string>
   </property>
  </action>
  <action name="actionMD4">
   <property name="text">
    <string>MD4</string>
   </property>
  </action>
  <action name="actionMD5">
   <property name="text">
    <string>MD5</string>
   </property>
  </action>
  <action name="actionSHA224">
   <property name="text">
    <string>SHA224</string>
   </property>
  </action>
  <action name="actionSHA256">
   <property name="text">
    <string>SHA256</string>
   </property>
  </action>
  <action name="actionSHA384">
   <property name="text">
    <string>SHA384</string>
   </property>
  </action>
  <action name="actionSHA512">
   <property name="text">
    <string>SHA512</string>
   </property>
  </action>
  <action name="actionSHA">
   <property name="text">
    <string>SHA</string>
   </property>
  </action>
  <action name="actionChiffrer_OpenPGP">
   <property name="text">
    <string>Encrypt Editor</string>
   </property>
  </action>
  <action name="actionDechiffrer_OpenPGP">
   <property name="text">
    <string>Decrypt Editor</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset>
     <normaloff>icones/open.png</normaloff>icones/open.png</iconset>
   </property>
   <property name="text">
    <string>Open txt file</string>
   </property>
  </action>
  <action name="actionSauver">
   <property name="icon">
    <iconset>
     <normaloff>icones/save.png</normaloff>icones/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionVider_Tampon">
   <property name="icon">
    <iconset>
     <normaloff>icones/empty.png</normaloff>icones/empty.png</iconset>
   </property>
   <property name="text">
    <string>Empty Editor</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="icon">
    <iconset>
     <normaloff>icones/copy.png</normaloff>icones/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="icon">
    <iconset>
     <normaloff>icones/paste.png</normaloff>icones/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCacher">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide buttons</string>
   </property>
  </action>
  <action name="actionAbout_GNUPG">
   <property name="text">
    <string>About GNUPG</string>
   </property>
  </action>
  <action name="actionKeymanager">
   <property name="text">
    <string>Key Manager</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>icones/cut.png</normaloff>icones/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionShow_text_icons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show text</string>
   </property>
  </action>
  <action name="actionChiffrer_un_fichier_Enigma">
   <property name="text">
    <string>Encrypt File (QPyCrypto)</string>
   </property>
  </action>
  <action name="actionD_chiffrer_un_fichier_Enigma">
   <property name="text">
    <string>Decrypt file (QPyCrypto)</string>
   </property>
  </action>
  <action name="actionImporter_une_cl_e">
   <property name="text">
    <string>Importer une clée</string>
   </property>
  </action>
  <action name="actionSigner_un_fichier">
   <property name="text">
    <string>Sign file</string>
   </property>
  </action>
  <action name="actionVerifier_signature">
   <property name="text">
    <string>Verify</string>
   </property>
  </action>
  <action name="actionChiffrer_un_fichier_Enigma_Test">
   <property name="text">
    <string>Chiffrer un fichier (Enigma Test)</string>
   </property>
  </action>
  <action name="actionD_chiffrer_un_fichier_Enigma_Test">
   <property name="text">
    <string>Déchiffrer un fichier (Enigma Test)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
